### PR TITLE
Web3raiju

### PR DESCRIPTION
**Pull request by Raiju2020**

### CODE REVIEW

File: css/spritual.wing.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+.pricing-content li del {
+    color:#aaa;
+}
+
+.price-bottom {
+    background:#f7f7f7;
+    text-align:center;
+    padding:25px 0;
+    border-radius: 0 0 4px 4px;
+}
```
Comment (divpaul2016) [inline css/spritual.wing.css]: are we using this ? if not remove it please .. might be cruft from the original

File: index.html [hunk 2]
```diff
@@ -69,7 +70,7 @@
             <li><a href="#service">Service</a></li>
             <li><a href="#doctor">Doctors</a></li>
             <li><a href="#facilities">Facilities</a></li>
-            <li><a href="#pricing">Pricing</a></li>
+            <li><a href="#pricing">Spiritual Wings</a></li>
```
Comment (divpaul2016) [inline index.html]: spiritual wings

File: index.html [hunk 3]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
-                <h2>Golden Glow Package</h2>
-                <h3>$65<sup>.99</sup> <span>/ one time</span></h3>
-                <ul>
-                  <li>Anthropometry (BMI, WH Ratio) <i class="fa fa-check icon-success"></i></li>
-                  <li>Post Examination Review <i class="fa fa-check icon-success"></i></li>
-                  <li>General Screening – Basic <i class="fa fa-check icon-success"></i></li>
-                  <li>Body Composition Analysis <i class="fa fa-check icon-success"></i></li>
-                  <li>GR Assessment & Scoring <i class="fa fa-check icon-success"></i></li>
-                </ul>
+          <!---end of St Pauls Men's League Sunday school begining-->
```
Comment (divpaul2016) [inline index.html]: Nice to comment the start of this section

File: index.html [hunk 4]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
+
+                  <div class="col-sm-3 pricing-box featured-price">
```
Comment (divpaul2016) [inline index.html]: name change

File: index.html [hunk 5]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
+                        </p>
+                        <br>
+                      </div>
+                    </div>
+                  </div>
 
-                <div class="price-bottom">
-                  <a href="#" class="btn btn-skin btn-lg">Purchase</a>
-                </div>
+          <!---end of  Sunday school  Samajam begining-->
```
Comment (divpaul2016) [inline index.html]: Start of Vanitha Samajam

File: index.html [hunk 6]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
+              <div class="spritual-content general">
+                <h3>St Mary's Women's League</h3>
+                  <h2>Secretary <span>: Mrs.Latha Goraphy</span></h2>
+                  <p>St. Mary’s Women’s League (Martha Mariam Samajam) is the key spiritual forum for the female members of St. George Church.
+                  The organization's mission is to enhance and enlighten the spiritual and social growth of women
+                  to promote Christian values within our families and the community at large</p>
+                <br>
               </div>
             </div>
           </div>
```
Comment (divpaul2016) [inline index.html]: end of vanitha samajam

File: index.html [hunk 7]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
-                  <li>Post Examination Review <i class="fa fa-check icon-success"></i></li>
-                  <li>General Screening – Regular <i class="fa fa-check icon-success"></i></li>
-                  <li><del>Health Screening Report</del> <i class="fa fa-times icon-danger"></i></li>
-                </ul>
+          <!---end of samajam,youth begining-->
 
-                <div class="price-bottom">
-                  <a href="#" class="btn btn-skin btn-lg">Purchase</a>
-                </div>
+          <div class="col-sm-3 pricing-box ">
```
Comment (divpaul2016) [inline index.html]: name to sedtion

File: index.html [hunk 8]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
     <section id="partner" class="home-section paddingbot-60">
       <div class="container marginbot-50">
         <div class="row">
           <div class="col-lg-8 col-lg-offset-2">
             <div class="wow lightSpeedIn" data-wow-delay="0.1s">
               <div class="section-heading text-center">
-                <h2 class="h-bold">Our partner</h2>
-                <p>Take charge of your health today with our specially designed health packages</p>
+                <h2 class="h-bold">Our Golden Feathers</h2>
+               <!-- <p>Take charge of your health today with our specially designed health packages</p>-->
```
Comment (divpaul2016) [inline index.html]: what are we doing here ?

File: css/spritual.wing.css [hunk 9]
```diff
@@ -0,0 +1,121 @@
+
+/* --- Pricing box --- */
```
Comment (divpaul2016) [inline css/spritual.wing.css]: spiritual section ??

File: stgeorge.iml [hunk 10]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (divpaul2016) [inline stgeorge.iml]: where are we using this file ??

Comment (Raiju2020) [reply stgeorge.iml]: I opened this file it seems, at the beginning of the week ,git asked me to add this before commit , so I did,  NO NEED TO MERGE THIS FILE, I  DON'T KNOW WHAT HAPPENED WHEN I OPENED

File: index.html [hunk 11]
```diff
@@ -595,79 +596,113 @@ <h2 class="h-bold">Health packages</h2>
...
+                  <div class="col-sm-3 pricing-box featured-price">
+                    <div class="wow bounceInUp" data-wow-delay="0.2s">
+                     <div class="spritual-content featured">
+
+                        <h3>St George Sunday School</h3>
+                        <h2>Principal <span>: Mr.Sujit Kuruvilla </span></h2>
+
+                        <p>12 pm to 1 pm after Holy Qurbana on Regular Sunday
+                          <br>
+                          Ms. Mili Ojus     -      Class 1
```
Comment (josephjoys) [inline index.html]: This must be in a <ul><li> tag . Try to make bulletted items

File: stgeorge.iml (hunk 10 above)
Comment (josephjoys) [reply stgeorge.iml]: This is created by intelliJ for your project no need to commit this. Can you add a .gitignore file in the repository then git will ignore  all *.iml files in future..

File: index.html (hunk 2 above)
Comment (divpaul2016) [reply index.html]: pricing needs to change to SpiritualWings

File: index.html (hunk 4 above)
Comment (divpaul2016) [reply index.html]: pricing-box featured-price

**Pull request closed without merging**